toqito/state_opt/bell_npa_constraints.py

```diff
@@ -0,0 +1,164 @@
+"""Generates NPA hierarchy constraints for Bell inequalities."""
+
+import cvxpy
+import numpy as np
+
+from toqito.state_opt.npa_hierarchy import Symbol, _gen_words, _reduce
```

Comment:
I wonder if putting this all in `npa_hierarchy.py` would be preferable and removing this file all together. Thoughts?